analyses/dg_points_to_plugin.cpp

```diff
@@ -436,7 +436,7 @@ std::string PointsToPlugin::pointsTo(llvm::Value* a, llvm::Value* b) {
 std::string PointsToPlugin::pointsToSetsOverlap(llvm::Value* a, llvm::Value* b) {
     assert(PTA);
     PSNode *nodeA = PTA->getPointsToNode(a);
-    PSNode *nodeB = PTA->getPointsToNode(a);
+    PSNode *nodeB = PTA->getPointsToNode(b);
```

Comment:
Could you put this change into a separate commit? It changes the behavior of the method. Thanks for finding it!

Reply:
Done!